src/main/java/mekanism/common/registration/impl/ItemDeferredRegister.java

```diff
@@ -54,6 +54,10 @@ public ITextComponent getDisplayName(@Nonnull ItemStack stack) {
         return register(name, () -> sup.apply(getMekBaseProperties()));
     }
 
+    public <ITEM extends Item> ItemRegistryObject<ITEM> registerUnburnable(String name, Function<Item.Properties, ITEM> sup) {
```

Comment:
The registerUnburnable method that already exists should probably be adjusted to just being `return register(name, Item::new);`

Reply:
do you mean `return registerUnburnable(name, Item::new);` or did i miss something?

Reply:
Yes, sorry I copy pasted and only adjusted the supplier -> method reference forgot to update the method it was calling.

Reply:
done.